src/ProofOfConcept.java

```diff
@@ -0,0 +1,173 @@
+import components.sequence.Sequence;
+import components.sequence.Sequence1L;
+import components.simplewriter.SimpleWriter;
+import components.simplewriter.SimpleWriter1L;
+
+public class ProofOfConcept {
+
```

Comment:
All of your methods are static. I'm not opposed to this, but it somewhat diverges from an OSU component. All the methods should be instance methods, and the sequence should be a variable at the top of your file.